AdobeIms/Controller/Adminhtml/OAuth/Callback.php

```diff
@@ -42,8 +42,8 @@ class Callback extends Action
      * RESPONSE_SUCCESS_CODE success code
      * RESPONSE_ERROR_CODE error code
      */
-    const RESPONSE_TEMPLATE = 'auth[code=%s;message=%s]';
-    const RESPONSE_REGEXP_PATTERN = 'auth\\[code=(success|error);message=(.+)\\]';
+    const RESPONSE_TEMPLATE = 'auth[code=%s;message=%s;name=%s]';
```

Comment:
Please use a separate request to retrieve user name (and potentially other user-related fields) after the successful authorization

Reply:
ok